### PR TITLE
Batch scripts now compile and run for scala REPL

### PR DESCRIPTION
**Pull request by RuMcClu**
also added MAX_CORRECTIONS to the constants file

### CODE REVIEW

File: airesources/Scala/hlt/Constants.scala [hunk 1]
```diff
@@ -34,15 +34,15 @@ object Constants {
   /** Number of turns it takes to dock a ship */
   val DOCK_TURNS = 5
 
-  /** Number of production units per turn contributed by each docked ship */
+  /** Number of turns it takes to create a ship per docked ship */
```
Comment (Janzert) [inline airesources/Scala/hlt/Constants.scala]: Might be a merge issue but, the previous phrase was correct. This is the production units produced per docked ship per turn, not the turns needed to produce a ship per docked ship.

**Comment by Snaar** [comment]
This definitely looks like bad merge. I'm afraid it would make sense to recreate this pull request with clean set of changes.

**Comment by Snaar** [comment]
DId you only intend to change run_game.bat and run_game.sh? Changes to those look clean, but rest looks like bad merge.

**Comment by Snaar** [comment]
Went ahead and fixed myself in fc284fc2a2ff6278449a6d24b85ffaa0f44b9963 and db94c510b9bc8c4c7bd15177ff97b1dcc87f46ef

Thanks for feedback!

**Pull request closed without merging**